devtools/conda-envs/test_env.yaml

```
@@ -24,7 +24,7 @@ dependencies:
 
     # Requirements for conversion tools
   - pyyaml
-  - ambertools >=18.0 # contains sufficiently recent ParmEd
+  - ambertools <=22.0 # contains sufficiently recent ParmEd
```

You might want to avoid solvers looking into silly old versions (18, etc.) by adding a lower constraint:

```suggestion
  - ambertools >=20,<=22.0
```

or, what I'd do:

```suggestion
  - ambertools =22.0
```

---

Yeah, I was thinking about it, I think it's best if we pin it to 22. THanks!